examples/tokens/ERC1155ownable.vy

```diff
@@ -26,6 +26,7 @@ paused: public(bool)
 
 # the contracts URI to find the metadata
 _uri: String[MAX_URI_LENGTH]
+_contractUri: String[MAX_URI_LENGTH]
```

Comment:
Was reading the section on the ERC1155 metadata and I think we need a bigger rewrite here.

Typically, a `baseURI` public state variable will be used to track the base of the `uri(id: uint256) -> String` function (although not strictly necessary)

Reply:
Dynamic URI is something I will work on next. I discovered the basic 1155 with static URI was missing the contract metadata functions. I've deployed this new version and minted a couple of items with it. Will dig up the url

Reply:
@fubuloubu I've added dynamic URI
it uses the base URI + String[78] for the uint2str output.